crates/bevy_render/src/texture/fallback_image.rs

```diff
@@ -54,15 +52,15 @@ fn fallback_image_new(
     image.texture_descriptor.usage |= TextureUsages::RENDER_ATTACHMENT;
 
     // We can't create textures with data when it's a depth texture or when using multiple samples
-    let texture = if format.describe().sample_type == TextureSampleType::Depth || samples > 1 {
+    let texture = if format.sample_type(None) == Some(TextureSampleType::Depth) || samples > 1 {
```

Comment:
Why is it ok to pass `None` here?

Reply:
Thank you for your sincere question. I don't know much about wgpu. These changes refer to [the API migration advice provided by changelog in wgpu 0.16](https://github.com/gfx-rs/wgpu/blob/v0.16/CHANGELOG.md#v0160-2023-04-19). 
If there is any inaccuracy here, please also ask Reviewer for advice.